TipCatDotNet.Api/Data/Models/PermissionLevel.cs

```diff
@@ -0,0 +1,9 @@
+namespace TipCatDotNet.Api.Data.Models
+{
+    public enum PermissionLevel: byte
```

Comment:
Why do we need this, if we have [HospitalityFacilityPermissions](https://github.com/tipcat-net/tipcat-net-api/pull/5/files?file-filters%5B%5D=.cs&file-filters%5B%5D=.csproj&file-filters%5B%5D=.json#diff-b6321859ad6051ba014798c822d6e7ba1a083f7e49bf7fbfe668d81765841ab1) already?